go.mod

```diff
@@ -1,12 +1,16 @@
-module github.com/upbound/upjet
+// SPDX-FileCopyrightText: 2023 The Crossplane Authors <https://crossplane.io>
+//
+// SPDX-License-Identifier: CC0-1.0
+
+module github.com/crossplane/upjet
 
 go 1.20
 
 require (
 	dario.cat/mergo v1.0.0
 	github.com/antchfx/htmlquery v1.2.4
-	github.com/crossplane/crossplane v1.10.0
-	github.com/crossplane/crossplane-runtime v0.20.0
+	github.com/crossplane/crossplane v1.13.2
```

Comment:
It seems that our crossplane dependencies are a bit old and updating them make sense to me. However, while switching to new architecture, protecting the versions may also be considered. What are your thoughts @jeanduplessis @ulucinar